examples/react-client/videoroom/src/RoomConnector.tsx

```diff
@@ -1,67 +1,73 @@
 import { Button } from "./Button";
 import { useConnect, useStatus, useDisconnect } from "./client";
 
+type FormValues = {
+  roomName: string;
+  userName: string;
+  roomManagerUrl: string;
+};
+
 export function RoomConnector() {
   const connect = useConnect();
   const status = useStatus();
   const disconnect = useDisconnect();
 
   const isUserConnected = status === "joined";
 
-  const connectToRoom = async ({
-    roomName,
-    username,
-    fishjamUrl,
-  }: Record<string, unknown>) => {
-    const res = await fetch(`${fishjamUrl}/room-manager/${roomName}/users/${username}`);
+  const connectToRoom = async ({ roomManagerUrl, roomName, userName }: FormValues) => {
+    localStorage.setItem("roomManagerUrl", roomManagerUrl);
+    localStorage.setItem("roomName", roomName);
+    localStorage.setItem("userName", userName);
```

Comment:
**suggestion:** we could move it to `handleSubmit` function, so `connectToRoom` is only responsible for connecting

Reply:
or even go for `persistValues` function and invoke it in `handleSubmit`